### PR TITLE
Update teams.txt

### PR DESCRIPTION
**Pull request by TonyScott44**

### CODE REVIEW

File: teams.txt [hunk 1]
```diff
@@ -4,8 +4,8 @@
 [ GitHub repository link ]
 [ Heroku/Deployment link ]
 
-Jason Paffenback, Anthony Mitchell, Taylor Yao
-One.Beer.Later.
-A revamp of our first project, we plan to add more functionality, user authentication, and new technologies. It will have similar look.
-https://github.com/jpaffenback/One.Beer.Later.
-https://one-beer-later.herokuapp.com/
+Angela Johnson,Sasi Koramutla,Tony Scott
+Viber
```
Comment (shaheinm) [inline teams.txt]: Viber is already an app name. (https://www.viber.com/)

File: teams.txt [hunk 2]
```diff
@@ -4,8 +4,8 @@
...
+Viber is an app for music artists and enthusiasts to meet each other through their shared interests in music. Users will input their musical preferences and meet other users with similar preferences by building themed playlists, rating others' playlists, rating artists and songs, and chatting with other users.
+https://github.com/AJ906/finalproject-ajskas
+https://viber-.herokuapp.com/
```
Comment (shaheinm) [inline teams.txt]: Please don't delete another group's information.  Add yours above or below theirs.

File: teams.txt (hunk 1 above)
Comment (TonyScott44) [reply teams.txt]: Yes, but this for educational purposes only, not looking to get legal rights to the name.

Comment (TonyScott44) [reply teams.txt]: We'll name it Vyber instead.

Comment (shaheinm) [reply teams.txt]: Vyber is fine, just make the change here in the PR where necessary.

**Pull request closed without merging**